client-app/main.ts

```diff
@@ -8,12 +8,14 @@ import Error403 from "./pages/403/403.vue";
 import "@fontsource/lato";
 import "@fontsource/roboto";
 import "@fortawesome/fontawesome-free/css/all.css";
+import SingInPage from "./pages/sign-in/sign-in-page.vue";
 import "../assets/static/styles/index.css";
 
 const router = createRouter({
   history: createWebHistory(),
   routes: [
     { path: "/", name: "Home", component: Home },
+    { path: "/signIn", name: "SignIn", component: SingInPage },

```

Comment:
The lowercase URL pattern is recommended (prefer /sign-in path instead of /signIn, etc).